docs/gateway-api/gateway-api-v1beta1.md

```diff
@@ -228,22 +232,32 @@ AKO accepts the following Gateway configuration for this release:
   2. Gateway MUST NOT contain protocols other than HTTP or HTTPS.
   3. Gateway MUST contain a hostname. Hostname as `*` is not supported and `*.domain` is supported.
   4. Gateway MUST NOT contain TLS modes other than `Terminate`.
+  5. If the Secret specified in Gateway-> listeners-> TLSConfig does not exist, our implementation will attach `SystemDefaultCert` to the Gateway. Once Secret is created, the `certRef` will be updated accordingly.
```

Comment:
`SystemDefaultCert` will be attached to Parent VS. Lets mention Avi Controller side object created corresponding to each Gateway object.